MessengerProj/src/main/res/values/strings.xml

```diff
@@ -204,23 +204,23 @@
     <!--permissions-->
     <string name="PermissionContacts">Access to contacts needed to connect with your friends across all your devices.</string>
     <string name="PermissionStorage">Storage access needed to send and save photos, videos, music, and other media.</string>
-    <string name="PermissionNoAudio">Microphone access needed to send voice messages.</string>
+    <string name="PermissionNoAudio">Mic access needed to send voice messages.</string>
```

Comment:
Is this abbreviation so common? I simply don't know :)

Reply:
I would argue yes, for the current userbase, which in turn is a sub-ideal argument.
The pictogram for microphone is universally recognized.